docs/graphql/core/guides/postgres/index.rst

```diff
@@ -55,3 +56,4 @@ The following are some Postgres features that can be used to manage your data an
   Functions <functions>
   Triggers <triggers>
   Indexes <indexes>
+  Import-data <import-data>
```

Comment:
Can we rename the file from `import-data.rst` to `import-csv-data.rst`. Maybe at some point we'll have a guide about importing data from a different source and then this will be confusing.

Reply:
@marionschleifer I will do that 🙂